### DISCUSSION TITLE
GitHub Release notes & soft line breaks

### DISCUSSION THREAD

**Discussion by oliviertassinari** [community/community#35750] [Other Feature Feedback, Questions, & Ideas]
The GitHub release notes behave strangely. Based https://github.github.com/gfm/#soft-line-breaks, shouldn't a soft line break in the markdown not create a line break in the HTML output? 

Say we take this input:

```
Line1.
Line2.

Line3.
```

Then GitHub will render it like this ✅

<img width="118" alt="Screenshot 2022-10-11 at 12 16 11" src="https://user-images.githubusercontent.com/3165635/195064402-d71620ef-c51f-4766-8c1f-9844a11b72e9.png">

https://github.com/oliviertassinari/bug-markdown-release/blob/main/CHANGELOG.md

But GitHub releases will render it like this ❌

<img width="66" alt="Screenshot 2022-10-11 at 12 16 21" src="https://user-images.githubusercontent.com/3165635/195064428-e092bb76-6367-4a36-9b72-4d8af900554d.png">

https://github.com/oliviertassinari/bug-markdown-release/releases/tag/v1.0.0

Why?

**Comment by jfullstackdev** [comment]
I reproduced the same, in Releases and here in GitHub Discussion, it will be line by line

but in README, it's not, same as you presented

as far as I can remember, yup, that's the old one and it's still the standard, so you need to at least make a tab for it to enter a new line

**Comment by jonas-schulze** [comment]
Inside a comment within a PR a softbreak is also rendered as a line break.
I am pretty sure that it used to be a simple space, which I would prefer.
I formed a habit of breaking lines at the end of sentences and most punctuations when writing Markdown or LaTeX,
such that git diffs remain small and show the exact part of a paragraph edited.
This habit now yields ugly text in GitHub comments. 😕

**Comment by trallnag** [comment]
Yeah, GitHub should really decide on one way to render Markdown.

**Comment by mickvangelderen** [comment]
Related to https://github.com/orgs/community/discussions/10981
